test/sanitize-field-names/fixtures.js

```diff
@@ -0,0 +1,309 @@
+'use strict'
```

Comment:
This filename should be prefixed with '_' as well to avoid being considered a file with tests.

Reply:
👍 changed